dev/com.ibm.ws.repository.parsers/build.gradle

```
@@ -0,0 +1,84 @@
+configurations {
+    anno_esa
+    jsp_esa
+    anno_metadata
+    jsp_metadata
+ }
+ 
+ dependencies {
+    anno_esa      'com.ibm.ws.repository.parsers.test:anno-esa:1.0.0'
+    jsp_esa       'com.ibm.ws.repository.parsers.test:jsp-esa:2.3.0'
+    anno_metadata 'com.ibm.ws.repository.parsers.test:anno-metadata:1.0.0'
+    jsp_metadata  'com.ibm.ws.repository.parsers.test:jsp-metadata:2.3.0'
+    
+ }
+
+task copyFirstAnnoEsaFileForUnittest(type:Copy) {
+    from configurations.anno_esa
+    into new File('test-resources/esa/')
+    rename 'anno-esa-*.*.*.*', 'com.ibm.websphere.appserver.anno-1.0.esa'
+}
```

```suggestion
task copyFilesForUnittest {
  copy {
    from configurations.anno_esa
    into new File('test-resources/esa/')
    rename 'anno-esa-*.*.*.*', 'com.ibm.websphere.appserver.anno-1.0.esa'
```
Update the other tasks to just be `copy {` inside of the task and the you only need to depend on one task.

---

Thank you!  That looks much cleaner.